src/main/java/com/google/api/codegen/transformer/FeatureConfig.java

```diff
@@ -38,7 +38,7 @@ public boolean enableMixins() {
 
   /** Returns true if streaming APIs are supported. */
   public boolean enableGrpcStreaming() {
-    return false;
+    return true;
```

Comment:
Since moving into mobile is a possibility, it is worth asking: Do Swift and Android Java have this support?